c/aws-c-common/yml.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
Why have this empty file? Note that it is called by the Makefile.

Reply:
Yeah, this script was used to generate the yml files and for some reason its content got lost. Maybe we could even add it at the top-level? It seems useful for others, too.